flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/restart/FixedDelayRestartStrategy.java

```diff
@@ -108,22 +87,7 @@ public static FixedDelayRestartStrategyFactory createFactory(Configuration confi
 			timeoutString
 		);
 
-		long delay;
-
-		try {
-			delay = Duration.apply(delayString).toMillis();
-		} catch (NumberFormatException nfe) {
-			if (delayString.equals(timeoutString)) {
-				throw new Exception("Invalid config value for " +
-					ConfigConstants.AKKA_WATCH_HEARTBEAT_PAUSE + ": " + timeoutString +
-					". Value must be a valid duration (such as '10 s' or '1 min')");
-			} else {
-				throw new Exception("Invalid config value for " +
-					ConfigConstants.EXECUTION_RETRY_DELAY_KEY + ": " + delayString +
-					". Value must be a valid duration (such as '100 milli' or '10 s')");
-			}
-		}
-
```

Comment:
Sorry for not being precise.
What I meant was that `ConfigConstants` used in exception handling in `RestartStrategyFactory` and in `FixedDelayRestartStrategy` are the same even though couple lines higher during configuration reading in both classes different `ConfigConstants` are used. It looks like removed exception handling was referring to wrong `ConfigConstants`? Or maybe wrong `ConfigConstant` is used in one case (`AKKA_WATCH_HEARTBEAT_INTERVAL` vs `AKKA_WATCH_HEARTBEAT_PAUSE`)? And probably in `FixedDelayRestartStrategy` constant `RESTART_STRATEGY_FIXED_DELAY_DELAY` should be used instead of `EXECUTION_RETRY_DELAY_KEY`? I'm not quite sure.